src/test/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/GenerateOrderCallbackHandlerTest.java

```diff
@@ -110,14 +112,50 @@ public void shouldPrepopulateFieldsOnAboutToStartEventIfNobodyObjectsCourt() {
         assertThat(response.getData()).contains(
             entry("directionList", ImmutableList.of("DOCUMENTS", "EYEWITNESS")),
             entry("docUploadDeadline", DEADLINE),
+            entry("docUploadForParty", "BOTH"),
             entry("eyewitnessUploadDeadline", DEADLINE),
+            entry("eyewitnessUploadForParty", "BOTH"),
+            entry("paperDetermination", "NO"),
+            entry("otherDirectionHeaders", "HEADER_UPLOAD"),
```

Comment:
Pls use Enum value to test instead of String wherever possible.

Reply:
It's a test, I'd rather have a string so that we additionally check the enum deserialise fine